### PR TITLE
Get rid of nose req

### PR DESCRIPTION
**Pull request by ufranske**

### CODE REVIEW

**Comment by Khrol** [comment]
Squash commits, please.

File: setup.py [hunk 1]
```diff
@@ -20,8 +20,5 @@
     include_package_data=True,
     author='Wargaming.NET',
     author_email='i_khrol@wargaming.net',
```
Comment (Khrol) [inline setup.py]: Maybe replace with `web_qa_auto@wargaming.net`?

File: grail/step_info.py [hunk 2]
```diff
@@ -60,7 +58,8 @@ def run_function(self):
     def get_description(self, output, result, exception_instance):
         message = ''
         if settings.export_mode:
-            assert_not_equal(result, StepResults.FAILED, 'Unexpected failure during export')
+            if result == StepResults.FAILED:
+                raise AssertionError('Unexpected failure during export')
```
Comment (Khrol) [inline grail/step_info.py]: `RuntimeError` would be more appropriate in this case.

File: setup.py (hunk 1 above)
Comment (ufranske) [inline setup.py]: Fixed.

File: grail/step_info.py (hunk 2 above)
Comment (ufranske) [inline grail/step_info.py]: Fixed.

**Pull request merged**